modules/create-user-workloads-aws-edge.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+//To-do: reintegrate installation-extend-edge-nodes-aws-local-zones.adoc with create-user-workloads-aws-edge.adoc. Requires global repo update of any xrefs/includes.
+
+:_mod-docs-content-type: PROCEDURE
+[id="create-user-workloads-aws-edge_{context}"]
+= Creating user workloads in an Outpost
+
+After you extend an {product-title} in an AWS VPC cluster into an Outpost, you can use edge compute nodes with the label `node-role.kubernetes.io/outposts` to create user workloads in the Outpost.
+
+.Prerequisites
+
+* You have extended an AWS VPC cluster into an Outpost.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+* You have created a compute machine set that deploys edge compute machines compatible with the Outpost environment.
+
+.Procedure
+
+. Configure a `deployment` resource file named `<application_deployment>.yaml` for an application that you want to deploy to the edge compute node in the edge subnet.
```

Comment:
I'd suggest mentioning a specific file name (as opposed to the `<substitute_this_word>` formatting) or removing the "named..." part of this step:

```suggestion
. Configure a `deployment` resource file for an application that you want to deploy to the edge compute node in the edge subnet.
```